README.md

```diff
@@ -115,12 +115,17 @@ nvm use 10 # must use node 8.11.1 and up because of N-API
 npm install --ignore-scripts
 ```
 
-Then clone and build [`prime_server`](https://github.com/kevinkreiser/prime_server#build-and-install).
+Then clone Valhalla
+
+```bash
+git clone -c core.symlinks=true --recurse-submodules https://github.com/valhalla/valhalla.git
```

Comment:
if i understand the docs correctly, this will duplicate the files that the symlinks point to right? in my opinion we dont want to do this on unix-like systems symlinks work perfectly fine there. would it be ok to call this out as required only for filesystems (ie NTFS or FAT) which do not support symlinks? What I would hate to happen is for us to end up in a scenario where a user makes a contribution that is an edit in one of the symlinks. In fact maybe it would be better to just remove the symlinks to avoid any future troubles with that? what do you think?

Reply:
> would it be ok to call this out as required only for filesystems (ie NTFS or FAT) which do not support symlinks?

It is a bit more subtle issue. If a user
- installed Git for Windows with symlink support enabled
- runs `git clone` with Administrator privileges
- runs `git clone` on local NTFS filesystem (not FAT)

then, `git clone` will create regular symlinks.  Details in this SO answer to [What happens when I clone a repository with symlinks on Windows?](https://stackoverflow.com/a/11664406/151641).

However, to keep things simple, I should indeed correct this change and add this note about `-c core.symlinks=true` for Windows users only.

> In fact maybe it would be better to just remove the symlinks to avoid any future troubles with that?

From perspective of a user building Valhalla from git repo on Windows, that would be best approach.

I can't count how many times got bitten by this isssue. I even have this sticky note to remember 😄 

```
Valhalla uses submodules and symlinks, so to clone correctly run as Administrator:

    git clone -c core.symlinks=true --recurse-submodules https://github.com/valhalla/valhalla.git

Verify symlinks to protobuf scripts have been replaced correctly

    cd D:\valhalla

    head -n 1 proto\fileformat.proto
    /** Copyright (c) 2010 Scott A. Crosby. <scott@sacrosby.com>

    but NOT this!

    head -n 1 proto\fileformat.proto
    ../OSM-binary/src/fileformat.proto
```

Reply:
Meanwhile, I have updated this change and moved the symlinks option to note for Windows users.